### DISCUSSION TITLE
The return result changed like this  \"faceFeatures\"

### DISCUSSION THREAD

**Discussion by yunchangxiaoguan** [fastapi/fastapi#7443] [Questions]
I use postman post a image get json result. but it add ' \\'in the key  \\"faceH\\": \\"136.88063\\", \\"faceW\\": \\"129.02844\\", \\"faceX\\": \\"1146.832\\", \\"faceY\\": \\"144.49599\\"}, {\\"faceFeatures\\"

how can i drop it

**Comment by tiangolo** [comment]
I imagine you solved your problem, thanks for closing the issue :+1:
